test/api/tutorials_test.rb

```diff
@@ -663,4 +663,30 @@ def test_tutorials_delete_incorrect_auth_token
     # Check authentication error
     assert_equal 419, last_response.status
   end
+
+#25: Testing for failure due to unauthorised account
+  # Delete a tutorial using unauthorised account
+  def test_student_cannot_delete_tutorial
+    # A user with student role which does not have permision to delete a tutorial
+    user = FactoryGirl.build(:user, :student)
+
+    # Number of tutorials before deletion
+    number_of_tutorials = Tutorial.count
+
+    # Tutorial to delete
+    tutorial_to_del = Tutorial.second
```

Comment:
Same here but you should be able to build a tutorial - though you may need to check the factory